R/guide-legend.R

```diff
@@ -18,6 +18,7 @@
 #'   guide overrides, and is combined with, the plot's theme.
 #' @param position A character string indicating where the legend should be
 #'   placed relative to the plot panels.
+     One of "top", "right", "bottom", "left", or "inside".
```

Comment:
This line needs a roxygen comment `#'` in front.

Reply:
Sure. I'll make the changes. I might close this one though and put in a new pull request though. I ran `devtools::document()` and there are are a lot of minor changes to the documentation for other functions, though I'll just add the relevant ones.